extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/FailedExecution.java

```diff
@@ -0,0 +1,47 @@
+package io.quarkus.scheduler;
+
+import java.time.Instant;
+
+/**
+ * This event is fired synchronously and asynchronously when an execution of a scheduled method throw an exception.
+ */
+public class FailedExecution {
+
+    // Keep the fields in order to minimize the breaking changes
+    public final String triggerId;
```

Comment:
Pls remove these fields. We kept them on `SkippedExecution` just for backwards compatibility. `ScheduledExecution` holds this information as well.